src/components/Cart/Cart.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ItemList from "../Item-list/Item-list";
+const cart = props => {
```

Comment:
to support standards, name your components with a starting capital letter so we are able to differentiate between functions and components.

Reply:
refactor this to use `map` instead of `for`.. break the operation into 2 steps.. the first step should multiply each item price by its quantity and the second step will add up all the items in the array. this can be 2 separate functions chained together. 

What i described above is functional programming 101